common/src/test/java/civitas/common/capabilityencryption/EncryptCapabilityTest.java

```diff
@@ -18,14 +18,16 @@ class EncryptCapabilityTest extends TestBase
 	EncryptCapability encryptCapability;
 
 	@Test
-	@DisplayName("creates an encrypted capability\n"
-			+ " - generates a reencrypt factor\n"
-			+ " - encrypts the capability associated with the context using the key and the reencrypt factor")
+	@DisplayName("""
+			creates an encrypted capability
+			 - generates a reencrypt factor
+			 - encrypts the capability associated with the context using the key and the reencrypt factor
+			""")
 	void test() {
-		CapabilityEncryption ENCRYPT_CAPABILITY_RESULT = new CapabilityEncryption(
+		CapabilityEncryption encryptCapabilityResult = new CapabilityEncryption(
```

Comment:
The reason for the all caps was that it should be moved to a TestData, it just did not happen.